lib/hotkeys/global-hotkeys.tsx

```
@@ -55,19 +57,23 @@ export function GlobalHotkeys() {
                     "[GlobalHotkeys] collapseSidebar hotkey triggered, but toggleSidebar function is not available. Is SidebarProvider an ancestor and context found?"
                 )
             }
-        }
+        },
+        openTradeComposer: () => setTradeComposerOpen('true')
     }
 
     const activeShortcuts = keyboardShortcuts.filter(
         (shortcut) => shortcut.scope === scope && handlers.hasOwnProperty(shortcut.id)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)